source/base/multithread_info.cc

```diff
@@ -152,19 +74,13 @@ MultithreadInfo::set_thread_limit(const unsigned int max_threads)
           n_max_threads = max_threads_env;
       }
   }
-  // Without restrictions from the user query TBB for the recommended number
-  // of threads:
-  if (n_max_threads == numbers::invalid_unsigned_int)
-    n_max_threads = tbb::task_scheduler_init::default_num_threads();
 
-  // Initialize the scheduler and destroy the old one before doing so
-  static tbb::task_scheduler_init dummy(tbb::task_scheduler_init::deferred);
-  if (dummy.is_active())
-    dummy.terminate();
-  dummy.initialize(n_max_threads);
```

Comment:
@tjhei I have removed the above snippet of code.

Reply:
This is required to set the number of threads TBB uses. Unless you want to delete TBB right now, I would leave it in.

Reply:
and this would be useful while we are comparing performance between TBB and taskflow

Reply:
Ah. I did not parse the initialization lines correctly. I have put back the initialization part.